src/main/java/org/opentripplanner/standalone/GraphBuilderParameters.java

```diff
@@ -116,6 +116,18 @@
      * Maximal distance between stops in meters that will connect consecutive trips that are made with same vehicle
      */
     public int maxInterlineDistance = 200;
+    
+    /**
+     * this field indicate the maximum size for island without stops
```

Comment:
nits: Capitalize first word.  Period at the end of the sentence. s/indicate/indicates/